jobs/openshift4/cluster/integreatly/install/Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+#!groovy
+@Library('delorean-pipeline-library')
+
+String cluserAPI = "https://api.${CLUSTER_DOMAIN}:6443"
+Boolean cleanup = params.CLEANUP
+
+
+node('cirhos_rhel7') {
+     timeout(60) { ansiColor('gnome-terminal') { timestamps {
+    //currentBuild.displayName = "${currentBuild.displayName} ${CLUSTER_DOMAIN}"
+    currentBuild.description = "cluster: ${CLUSTER_DOMAIN}\n"
+
+
+    stage('Login to OpenShift 4 cluster') {
+        sh "oc login ${cluserAPI} --insecure-skip-tls-verify=true -u ${ADMIN_USERNAME} -p ${ADMIN_PASSWORD}"
+    } 
+
+    stage('Clone Integreatly Operator') {
+        dir('integreatly-operator') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: "${INTEGREATLY_OPERATOR_BRANCH}"]],
+                doGenerateSubmoduleConfigurations: false,
+                extensions: [],
+                userRemoteConfigs: [[url: "${INTEGREATLY_OPERATOR_REPOSITORY}"]]
+            ])
+            sh "git clean -xdf"
+        } 
+    }
+
+    stage('Clean up') {
+        when (params.CLEANUP){
+            println "Cleaning..."
+            dir("integreatly-operator") {
+                sh "make cluster/cleanup/olm"
+                sh "make cluster/cleanup/crds" 
+            }
+        }
+    }
+
+    stage('Create Integreatly Operator') {
+        sh "export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}"
+        sh "export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}"
```

Comment:
We should be using a jenkins credential for these and use withCredentials see https://github.com/integr8ly/ci-cd/blob/master/jobs/openshift4/cluster/deprovision/Jenkinsfile#L17-L29

Reply:
so I only need use that piece of code and I don't need any aws job attributes etc.?

Reply:
Leave this for the moment, we will create a follow up task to fix this

Reply:
We will come back to this and fix it in a follow up issue